cli/src/bin/phylum.rs

```diff
@@ -103,29 +130,9 @@ async fn handle_commands() -> CommandResult {
     // We initialize these value here, for later use by the PhylumApi object.
     let timeout = matches.get_one::<String>("timeout").and_then(|t| t.parse::<u64>().ok());
 
-    // Check for updates, if we haven't explicitly invoked `update`.
-    //
-
-    if matches.subcommand_matches("update").is_none() {
-        let now = UNIX_EPOCH.elapsed().expect("Time went backwards").as_secs() as usize;
-
-        let check_for_updates = config.last_update.map_or(true, |last_update| {
-            const SECS_IN_DAY: usize = 24 * 60 * 60;
-            now - last_update > SECS_IN_DAY
```

Comment:
Since `last_update` comes from the config, there was a chance this could underflow if somebody threw a big number in there (e.g., maybe just trying to disable the update check). I switched this to a `.saturating_sub()` so that it evaluates to 0 if `last_update > now`.